### PR TITLE
[OFFAPPS-626] CSS Fix

### PR DESCRIPTION
**Pull request by pmgrove**
:koala:

added overflow:auto to css to fix long email addresses flowing over the boundary of app

Before: 
before

After:
after

/cc @zendesk/vegemite @zendesk/wombat 

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-626

### Risks
 - None

### CODE REVIEW

**Comment by maximeprades** [comment]
:+1:

**Comment by sandlerr** [comment]
:+1:

**Comment by danielbreves** [comment]
Why not `word-wrap`?

File: app.css [hunk 1]
```diff
@@ -196,6 +196,7 @@
         color: #999;
         font-size: 11px;
         margin-bottom: 3px;
+        overflow: auto;
```
Comment (danielbreves) [inline app.css]: Why not `word-wrap` instead?

**Comment by ocke** [comment]
:+1:

File: app.css (hunk 1 above)
Comment (pmgrove) [inline app.css]: yeah, good call, ```word-wrap``` does work better. Will update.

**Comment by pmgrove** [comment]
@danielbreves :haircut:

**Comment by danielbreves** [comment]
:+1:

**Pull request merged**